auth/src/main/kotlin/com/hedvig/android/auth/storage/AuthTokenStorage.kt

```diff
@@ -0,0 +1,47 @@
+package com.hedvig.android.auth.storage
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.stringPreferencesKey
+import com.hedvig.authlib.AccessToken
+import com.hedvig.authlib.RefreshToken
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+// TODO Work with expiration dates.
```

Comment:
Not needed yet since we'll just rely on 401 to know when to retry. If we want to do some preemptive fetches and so on later on we can expand this,